packages/core/src/typeschema/types.ts

```diff
@@ -97,6 +96,15 @@ const PROFILE_SCHEMAS_BY_URL: { [profileUrl: string]: InternalTypeSchema } = Obj
 // is maintained per profile URL
 const PROFILE_DATA_TYPES: { [profileUrl: string]: DataTypesMap } = Object.create(null);
 
+// Special case names for StructureDefinitions that are technically "profiles", but are used as base types.
```

Comment:
What's the algorithm for determining this set of special cases? Could you add that as a comment?

Reply:
Right now it's 100% manual.  Yes, will add comments here on the philosophy.

Reply:
Fixed ✅